internal/controller/k0smotron.io/k0smotroncluster_configmap.go

```diff
@@ -36,6 +36,12 @@ func init() {
 }
 
 func (r *ClusterReconciler) generateCM(kmc *km.Cluster) (v1.ConfigMap, error) {
+	// If LB type svc, force ports to 6443 and 8132
```

Comment:
Why do we force it? I was under the impression, that we can still use custom ports, can't we?

Reply:
The idea was that when LB svc is in front, we can allow k8s to pick random node ports instead of users having to worry about which port is free etc.

Reply:
Removed the forcing of the ports now.